atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/service/SimpleTransactionService.java

```diff
@@ -24,15 +24,15 @@
 import com.palantir.atlasdb.keyvalue.api.KeyValueService;
 import com.palantir.atlasdb.keyvalue.api.Value;
 import com.palantir.atlasdb.transaction.encoding.TimestampEncodingStrategy;
-import com.palantir.atlasdb.transaction.encoding.V1EncodingStrategy;
 import com.palantir.atlasdb.transaction.impl.TransactionConstants;
 
 public class SimpleTransactionService implements EncodingTransactionService {
     private final KeyValueService keyValueService;
-    private final TimestampEncodingStrategy encodingStrategy = V1EncodingStrategy.INSTANCE;
+    private final TimestampEncodingStrategy encodingStrategy;
 
-    public SimpleTransactionService(KeyValueService keyValueService) {
+    public SimpleTransactionService(KeyValueService keyValueService, TimestampEncodingStrategy encodingStrategy) {
```

Comment:
This will require us to update the LIP - it's fine, but we should be aware of this.